File: C/glicose.c
#include <stdio.h>
#include <locale.h>

double glicose;

int main() {
    setlocale(LC_ALL, "Portuguese");

    printf("Digite a média da glicose: ");
    scanf("%lf", &glicose);

    if (glicose <= 100) {
        printf("Classificação: normal");
    }
    else if (glicose <= 140) {
        printf("Classificação: elevado");
    }
    else {
        printf("Classificação: diabetes");
    }

}

File: Java/glicose.java
import java.util.Locale;
import java.util.Scanner;

public class glicose {

    public static void main(String[] args) {

        Locale.setDefault(Locale.US);
        Scanner sc = new Scanner(System.in);

        System.out.print("Digite a medida da glicose: ");
        double glicose = sc.nextDouble();

        String classificacao;

        if (glicose <= 100) {
            classificacao = "normal";
        }
        else if (glicose <= 140) {
            classificacao = "elevado";
        }
        else {
            classificacao = "diabetes";
        }

        System.out.printf("Classíficação: %s", classificacao);

        sc.close();

    }
}
